ui/admin/tests/integration/components/ordered-series-diagram/index-test.js

```diff
@@ -0,0 +1,78 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'admin/tests/helpers';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module(
+  'Integration | Component | ordered-series-diagram/index',
+  function (hooks) {
+    setupRenderingTest(hooks);
+
+    test('it renders two icon series', async function (assert) {
+      assert.expect(4);
+      await render(hbs`
+      <OrderedSeriesDiagram as |d|>
+        <d.item @icon='user' @arrow={{true}}>
+          Client
+        </d.item>
+        <d.item @icon='settings' >
+          Egress Worker
+        </d.item>
+      </OrderedSeriesDiagram>
+    `);
```

Comment:
This looks like lines 14-22 need to be indented in one more.

Reply:
@cameronperera got will take care of it

Reply:
+1 to this suggestion.